Documentation/Architecture/Package_Manager/Package_Manager.md

```diff
@@ -8,11 +8,11 @@ Package vetting for showing up in the package manager UI should be done in multi
 
 The main goal is that all packages are contained within the Mixed Reality Toolkit Repo, or some separate repo mantained by the same community. This brings clarity to users about where issues can be filed, where to find more information about specific packages, and how they can get their packages added. This will explicitly not display any packages that are not contained within these constraints; while they may be good packages that work well with Mixed Reality Toolkit, they are not mantained within the community and will not be promoted in the same way. Instead, it will be up to the package owners to promote it's use in another way. We may find other promotion techniques if we as a community believe a package provides value but is not part of our ecosystem. 
 
-### Microsoft Packages
+### Release Packages
 
-Microsoft packages are promoted and mantained by Microsoft, with some promise of stability and support (TBD). These will be packages like Azure integration that we believe work well with Mixed Reality Toolkit and explicitly provide features that Microsoft is promoting. These packages will also have a strict process for submitting code changes.
+Release packages are promoted and mantained with some promise of stability and support (TBD). These will be packages like Azure integration that we believe work well with Mixed Reality Toolkit and explicitly provide features that Microsoft is promoting. These packages will also have a strict process for submitting code changes.
 
-### Community Packages
+### Preview Packages
```

Comment:
preview doesn't feel quite right here. these are more like unity asset store packages rather than preview features. most (?) packages in this category are likely not going to be appropriate to move to the "Release" category

Reply:
we should also create a category for experimental features.

Reply:
What's the difference between experimental and (whatever community/Preview changes to)?

Reply:
experimental is for features that are being test flighted that may never ship and have no promise to be functional

Reply:
I think we've landed on nuget being our solution, so lets align with common terminology from there. "Prerelease" should be as you described experimental above.